4.0/init.sh

```diff
@@ -21,6 +21,11 @@ else
   echo "Skipping optional US postcode import"
 fi;
 
+if [ "$IMPORT_US_TIGER" = "true" ]; then
```

Comment:
Looks like we are going to merge https://github.com/mediagis/nominatim-docker/pull/293 pretty soon, so we can remove that from that PR. What do you think?

Reply:
PR is merged now, please take that into consideration for your PR, thanks!